js/introModal.js

```diff
@@ -12,6 +12,25 @@ function triggerIntroModal(){
     // is visiting the page for the first time.
     if(!pageSeen){
 
+
+        tour = new Shepherd.Tour({
```

Comment:
I thought the idea was that we keep the intro modal, and add a button there to start the tour?